src/twisted/python/context.py

```diff
@@ -124,6 +124,13 @@ def getContext(self, key, default=None):
         return self.currentContext().getContext(key, default)
 
 
+
+theContextTracker = ThreadedContextTracker()
+call = theContextTracker.callWithContext
+get = theContextTracker.getContext
+
+
+
 def installContextTracker(ctr):
```

Comment:
In theory this is public API. I think that this should have a .removal release note to inform that this method was removed.
I don't think that this needs the deprecation process but please also send an email to the mailing list to let anyone know that this will be removed and see if there are any objections.
After 2 weeks this can be merged.

Maybe this API is important..so better to check with the others.

Thanks

Reply:
I did not remove `installContextTracker` in this PR

Reply:
True... Sorry. I was testing the GitHub mobile app for this review. My bad.
In this case, i think that installContextTracker needs to be deprecated here.
I don't see it used anywhere else and I think that it should be removed.
Thanks!

Reply:
Can I merge this as-is and deprecate installContextTracker() in a separate PR?  @hawkowl wrote this.  I'm not sure if there are other use cases for using this that are outside the Twisted codebase.